spec/requests/pseuds_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe "Pseuds", type: :request do
```

Comment:
Can you [leave out `type: :request`](https://github.com/otwcode/otwarchive/blob/f5e775b6e87e822e03baa67abd9e9e516e4ab7f3/spec/spec_helper.rb#L120)?